dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/BaseClasses.mo

```diff
@@ -25,20 +30,11 @@ package BaseClasses
 
   */
 
-    import Modelica;
-    import Dynawo.Connectors;
-    import Dynawo.Electrical.Controls.Basics.SwitchOff;
-    extends SwitchOff.SwitchOffDCLine;
-
     Connectors.ACPower terminal1 (V(re(start = u10Pu.re), im(start = u10Pu.im)), i(re(start = i10Pu.re), im(start = i10Pu.im))) annotation(
       Placement(visible = true, transformation(origin = {-100, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {-100, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
     Connectors.ACPower terminal2 (V(re(start = u20Pu.re), im(start = u20Pu.im)), i(re(start = i20Pu.re), im(start = i20Pu.im))) annotation(
       Placement(visible = true, transformation(origin = {100, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {100, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
 
-    parameter Types.ReactivePowerPu Q1MinPu  "Minimum reactive power in p.u (base SnRef) at terminal 1";
-    parameter Types.ReactivePowerPu Q1MaxPu  "Maximum reactive power in p.u (base SnRef) at terminal 1";
-    parameter Types.ReactivePowerPu Q2MinPu  "Minimum reactive power in p.u (base SnRef) at terminal 2";
-    parameter Types.ReactivePowerPu Q2MaxPu  "Maximum reactive power in p.u (base SnRef) at terminal 2";
     parameter Real KLosses "Coefficient between 0 and 1 (no loss) modelling the losses in the HVDC";
```

Comment:
I think comment for KLosses is not clear..

Reply:
What do you think of  : "Losses coefficient between 0 and 1 : 1 if no loss in the HVDC link, < 1 otherwise"

Reply:
perfect, thanks!